discord/role.py

```diff
@@ -167,8 +198,9 @@ class Role(Hashable):
             operators on the role objects themselves.
 
     managed: :class:`bool`
-        Indicates if the role is managed by the guild through some form of
-        integrations such as Twitch.
+        Indicates if the role is managed by the guild.
+        This is true if any of :meth:`Role.is_integration`, :meth:`Role.is_premium_subscriber`,
+        or :meth:`Role.is_bot_managed` are ``True``.
```

Comment:
is there a chance this is also true for monetization stuff

Reply:
I actually thought the same but as said before I could not test that. If anyone could check this and tell it would be appreciated.

Reply:
It *is* in fact `True`, but because of `Role.is_integration` as I explained in my later comment, so this should be fine.